apps/sparo-lib/assets/sparo-profile.json

```
@@ -34,6 +36,10 @@
   /**
    * A list of arbitrary additional folders to be included for checkout,
    * not necessarily corresponding to any workspace project.
+   * Each path given should be relative to the root directory of the Monorepo.
+   * Wildcards and glob patterns are not permitted due to a number of shortcomings.
+   * See the git sparse-checkout documentation for details:
+   * https://git-scm.com/docs/git-sparse-checkout#_internalsnon_cone_problems
```

```suggestion
   * The paths should use forward slashes, without a leading slash, and should be to the 
   * root folder of the monorepo.  Wildcards and glob patterns are not supported for
   * performance reasons.
```

I feel like "cone mode" is too 🤓, and a major benefit of Sparo is never having to understand that concept. We do mention "cone mode" on the website, but mainly to highlight how confusing Git can be without Sparo.